docs/IR.md

```diff
@@ -309,7 +309,7 @@ More details can be found in [External Data](ExternalData.md).
 
 There are two official ONNX variants; the main distinction between the two is found in the supported types and the supported operators.
 
-With respect to supported types, the __ONNX__ definition recognizes only tensors as input and output types, while the Classical Machine Learning extension __ONNX-ML__ also recognizes sequences and maps.
+With respect to supported types, both __ONNX__ and __ONNX-ML__ definition recognize tensors, sparse tensors, sequences, maps, and optionals as input and output types.
```

Comment:
I think it would be good to clarify IR-version distinctions: that is, optional type is supported from IR-version X, etc.

Reply:
Even release version/IR-version etc.